nav2_theta_star_planner/include/nav2_theta_star_planner/theta_star.hpp

```diff
@@ -185,7 +188,7 @@ class ThetaStar
   bool isSafe(const int & cx, const int & cy, double & cost) const
   {
     double curr_cost = getCost(cx, cy);
-    if (curr_cost < LETHAL_COST) {
+    if ((costmap_->getCost(cx, cy) == UNKNOWN_COST && allow_unknown_) || curr_cost < LETHAL_COST) {
```

Comment:
not sure if a different cost should be considered for the planning itself when the cell is unknown rather than using the maximum (which if I understand correctly would penalize unknown tiles the same way as obstacles)

Reply:
Look at what NavFn does with this value and you can copy that methodology here

Reply:
just to confirm, that is [setting it to the obstacle cost - 1](https://github.com/ros-planning/navigation2/blob/main/nav2_navfn_planner/src/navfn.cpp#L266)?